doc/conf.py

```diff
@@ -233,6 +233,7 @@
             'reference/xarray/line': 'ref/api/manual/hvplot.hvPlot.line',
             'reference/xarray/quadmesh': 'ref/api/manual/hvplot.hvPlot.quadmesh',
             'reference/xarray/rgb': 'ref/api/manual/hvplot.hvPlot.rgb',
+            'reference/xarray/barbs': 'ref/api/manual/hvplot.hvPlot.barbs',
```

Comment:
There's no need for this change, this is only to create redirects for old pages.